### DISCUSSION TITLE
Connect exiting organization to Education account

### DISCUSSION THREAD

**Discussion by scott-beaudry** [community/community#140619] [GitHub Education]
### Hi everyone,

Hello, we've recently received our Educational licensing, but our members don't have any of the features on their account. We had an existing organization prior to receiving our Educational status, can the old organization be tied to this?

I've reached out to the GitHub support email, but have not yet heard back.

**Comment by itisrude** [comment]
To connect an existing organization to your GitHub Education account, you'll need to reach out to GitHub support to associate the educational benefits with your existing organization. If you've already reached out but haven't heard back, it's worth double-checking that you submitted the request through GitHub's Education support form and included relevant details (such as your organization name and proof of educational status). 

If you're waiting, consider contacting them again or looking for the status of your ticket. It might also help to ensure that your GitHub Education benefits are active and associated with the correct email/account.
